### ISSUE TITLE
what is the use of this part?

### ISSUE THREAD

**Issue by rishabhsingh2** [markfoodyburton/ubond#2]
............................................................................................................................................
# Remote control can be setup on UNIX socket
# and TCP / HTTP protocol.
# remote control will output statistics only at the moment.
# You can visualise statistics using the www/ templates.
# RAW json stream available on http://[control_bind_host]:[control_bind_port]/status
# Edit www/js/ubond.js url to point to your ubond status URL.
#control_unix_path = "@localstatedir@/run/ubond.sock"
#control_bind_host = "0.0.0.0"
#control_bind_port = "1040"
.............................................................................................................................................

what s the use of this part?
and 
control_unix_path = "@localstatedir@/run/ubond.sock" 
where present this path ?
please help!

**Comment by markfoodyburton** [comment]
I use it to collect stats and graph them.

**Comment by rishabhsingh2** [comment]
> I use it to collect stats and graph them.

but where is present control_unix_path.

**Comment by markfoodyburton** [comment]
I set mine to 
control_unix_path = "/var/run/ubond.sock"

**Comment by rishabhsingh2** [comment]
i set mine to
#control_unix_path = "/home/debian/mlvpn-bin/var/run/mlvpn-sock"
but getting error if run the mlvpn.....
2020-09-08T05:59:38 [INFO/config] new password set
2020-09-08T05:59:38 [INFO/config] reorder_buffer_size changed from 0 to 64
2020-09-08T05:59:38 [INFO/config] P1l1r1 tunnel added
2020-09-08T05:59:38 [INFO/config] P1l1r2 tunnel added
2020-09-08T05:59:38 [INFO] created interface `Peer1'
..........................................................................................................................................................................................
2020-09-08T05:59:38 [WARN/control] cannot bind socket /home/debian/mlvpn-bin/var/run/mlvpn-sock/: No such file or directory
.......................................................................................................................................................................................................
2020-09-08T05:59:38 [INFO] P1l1r1 bind to 10.100.1.20
2020-09-08T05:59:38 [INFO] P1l1r2 bind to 10.100.1.20

**Comment by markfoodyburton** [comment]
Does the directory exist, is it writable?

**Comment by rishabhsingh2** [comment]
yes directory exist,yes it is writable. but what write in inside directory?

**Comment by markfoodyburton** [comment]
"/home/debian/mlvpn-bin/var/run/mlvpn-sock/: No such file or directory" dont understand why you are trying to write to a directory here. It should just be a file.

**Comment by rishabhsingh2** [comment]
yes , mlvpn-sock is a file not a directory. i don't  why getting no such file or directory.

**Comment by rishabhsingh2** [comment]
sir, response???

**Comment by markfoodyburton** [comment]
I dont know either
